lunar/distribution.yaml

```diff
@@ -1063,6 +1063,21 @@ repositories:
       url: https://github.com/ros-perception/perception_pcl.git
       version: lunar-devel
     status: maintained
+  pid:
+    doc:
+      type: git
+      url: https://bitbucket.org/AndyZe/pid.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
```

Comment:
yeah I figured we would notice for existing entries, I was just wondering in conceptually there can be a reason to override this. But you're right that if it passes for all existing entries I think it's safe to enforce it and to deal with the corner case if one appears someday.